tests/math/test_jax_box.py

```diff
@@ -1,167 +0,0 @@
-# Copyright 2018-2020 Xanadu Quantum Technologies Inc.

```

Comment:
We nuked a lot of tests here and didn't and many more. Are we sure our current test coverage satisfies our consistency needs?

Reply:
There was a _lot_ of redundancy in the existing test suite. The `test_jax_box` file included tests for JAX matmul, conj, etc., that were simply repeated in `test_fn.py`. The remaining tests in this file simply tested the TensorBox machinery, which is now (happily) gone